src/lib/hover_thrust_estimator/ZeroOrderHoverThrustEkf.hpp

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <matrix/matrix/math.hpp>
+#include <ecl/geo/geo.h>
+
+class ZeroOrderHoverThrustEkf
+{
+public:
+	ZeroOrderHoverThrustEkf() = default;
+	~ZeroOrderHoverThrustEkf() = default;
+
+	void predict(float _dt);
+	void fuseAccZ(float acc_z, float thrust);
+	void setProcessNoise(float process_noise) { _Q = process_noise; }
+	void setMeasurementNoise(float measurement_noise) { _R = measurement_noise; }
+
+	float getHoverThrustEstimate() const { return _hover_thr; }
+private:
+	float _hover_thr{0.5f};
+
+	float _gate_size{3.f};
+	float _P{0.2f};
+	float _Q{0.01f};
+	float _R{0.5f};
+	float _dt{0.02f};
+
+	float computeH(float thrust);
```

Comment:
A lot of these functions could be marked as const

Reply:
Yep